### DISCUSSION TITLE
Settings to use for lg c1?

### DISCUSSION THREAD

**Discussion by ThetaX55** [community/community#120150] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I got a 3070 nvidea card. SO it has the HDMI 2.1 standards.
What are the best settings to use.
Maybe a file to just load the one's you people have. 
I see you can put settings per game as well.
A feature to load someone else's profile would be nice if not available. IDK.
I pretty much used this for auto dim feature turn off. I followed a youtube video.

**Comment by ThetaX55** [comment]
I am using rratings settings from the site. Seem ok.
i turned off autodim because it got annoying wile web browsing.

**Comment by ebndev** [comment]
Hi @ThetaX55,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try forums that focus on LGs / TVs, like possibly finding a subreddit!

The GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
